src/test/store/receive-profile.test.js

```diff
@@ -17,7 +17,6 @@ import {
   _fetchProfile,
 } from '../../actions/receive-profile';
 
-import preprocessedProfile from '../fixtures/profiles/profile-2d-canvas.json';
```

Comment:
This was the only place using this, and I was basing my tests on this code block. I think this justifies its removal in this PR.